device/cuda/src/seeding/seed_finding.cu

```diff
@@ -162,28 +172,42 @@ vecmem::data::vector_buffer<seed> seed_finding::operator()(
     const sp_grid_const_view& g2_view,
     const std::vector<unsigned int>& grid_sizes) const {
 
-    // Get the prefix sum for the spacepoint grid using buffer.
-    const device::prefix_sum_t sp_grid_prefix_sum = device::get_prefix_sum(
-        grid_sizes, (m_mr.host ? *(m_mr.host) : m_mr.main));
+    // Create prefix sum of grid_sizes vector
+    std::vector<unsigned int> grid_sizes_sum(grid_sizes.size());
```

Comment:
`std::vector` works here, but I would still rather use `vecmem::vector`. That way, when we use a single (shared/managed) memory resource, we could skip the explicit copy later on.